tests/com/twitter/intellij/pants/integration/OSSProjectInfoResolveTest.java

```diff
@@ -63,6 +65,15 @@ public void testTargetType() {
     assertFalse(greetTarget.isScalaTarget());
   }
 
+  public void testAvailableTargetTypes() {
+    final ProjectInfo info = resolveProjectInfo("examples/src/scala/org/pantsbuild/example/hello/");
+
+    final List<String> availableTargetTypes = Arrays.asList(info.getAvailableTargetTypes());
+    assertNotNull(availableTargetTypes);
```

Comment:
not null should be asserted on `info.getAvailableTargetTypes()` instead of `availableTargetTypes`, or you can remove this assert probably, because `Arrays.asList` will probably fail on null anyway.

Reply:
Removed that and added `@NotNull` annotation - I think Gson just defaults to an empty array but just to be on thesafe side.